src/screen/EditGestureScreen.js

```diff
@@ -0,0 +1,392 @@
+              <EditGestureActionBox>
+                <EditGestureListText>Custom Gesture</EditGestureListText>
+              </EditGestureActionBox>
+              <EditGestureFunctionBox>
+                <Picker
+                  mode="dropdown"
+                  style={{ width: 150 }}
+                  dropdownIconColor="#7e94ae"
+                  selectedValue={selectedLanguage}
+                  onValueChange={(itemValue, itemIndex) => {
+                    axios.post(
+                      `${SERVER_PORT}/users/${
+                        JSON.parse(idToken.current).user.email
+                      }/customGesture`,
+                      { function: itemValue },
+                    );
+
+                    setSelectedLanguage(itemValue);
+                  }}
+                >
+                  <Picker.Item label="없음" value="" />
+                  <Picker.Item
+                    label="브라우저 뒤로가기"
+                    value="goBackInBrowser"
+                  />
+                  <Picker.Item
+                    label="브라우저 앞으로가기"
+                    value="goForwardInBrowser"
+                  />
+                  <Picker.Item
+                    label="브라우저 탭 앞으로가기"
+                    value="goForwardInTap"
+                  />
+                  <Picker.Item
+                    label="브라우저 탭 뒤로가기"
+                    value="goBackInTap"
+                  />
+                  <Picker.Item label="볼륨 업" value="volumeUp" />
+                  <Picker.Item label="볼륨 다운" value="volumeDown" />
+                  <Picker.Item label="음소거" value="mute" />
+                  <Picker.Item label="재생" value="paly" />
+                  <Picker.Item label="일시정지" value="pause" />
+                </Picker>
+              </EditGestureFunctionBox>
+              <Ionicons name="refresh" size={15} color="#999999" />
+            </EditGestureListBox>
+          </ScrollView>
+        </View>
+      </EditGestureTextBox>
+    </EditGestureContainer>
+  );
+};
+
+const EditGestureContainer = styled.View`
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  background-color: #f3eee6;
+`;
+
+const EditGesturePreviousScreenButton = styled.TouchableOpacity`
+  position: absolute;
+  top: 50px;
+  left: 20px;
+`;
+
+const EditGestureAnimationBox = styled.View`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 70px;
+  height: 280px;
+  width: 280px;
+  border: 2px solid #666666;
+  border-radius: 10px;
+  z-index: -1;
+`;
+
+const Box = styled.View`
+  margin: 15px 15px;
+  width: 30px;
+  height: 30px;
+  background-color: #7e94ae;
+  border-radius: 20px;
+`;
+
+const EditGestureTextBox = styled.View`
+  display: flex;
+  justify-content: center;
+  align-self: center;
+`;
+
+const EditGestureTitleBox = styled.View`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  padding: 0 60px;
```

Comment:
```suggestion
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
  padding: 0 60px;
```
css에 대해서 정확히 알지는 못하지만 flex 속성을 주고 바로 밑에 flex-direction을 적어주는게 뭔가 한눈에 파악하기에 더 좋을 것 같다는 의견입니다.